File: Java/Car.java
class Car {
    Integer id;
    String license;
    Account driver;
    int passengers;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void dataCar() {
        System.out.println("Lisence: " + license + " Name Driver: " + driver.name);
    }
}

File: JS/Car.js
function Car(lisence, driver) {
  this.id;
  this.lisence = lisence;
  this.driver = driver;
  this.passengers;
}
Car.prototype.printDateCar = function (){
  
  console.log(this.driver)
  console.log(this.driver.name)
  console.log(this.driver.document)
  
}